### DISCUSSION TITLE
Does "EU(>2023)" mean 2023 and after, or 2024 and after or what?

### DISCUSSION THREAD

**Discussion by henu** [Hyundai-Kia-Connect/kia_uvo#863] [Q&A]
Hello!

I live in EU and have Ioniq 5 that is manufactured in 2023 (in August if I can remember precisely), and I'm not sure to which column it goes. EU or EU(>2023)?

I noticed there are some data missing in latter column, so if my car is considered EU(>2023), I would be happy to test those features.

**Comment by Esprit1st** [comment]
>2023 should mean 2024 and newer, otherwise it should be >2022 or >=2023 ... Technically.On Apr 12, 2024 05:36, henu ***@***.***> wrote:
Hello!
I live in EU and have Ioniq 5 that is manufactured in 2023 (in August if I can remember precisely), and I'm not sure to which column it goes. EU or EU(>2023)?
I noticed there are some data missing in latter column, so if my car is considered EU(>2023), I would be happy to test those features.

**Reply by henu** [reply]
Yeah, good point. On the other hand, does it mean "greater than the first day of 2023"? :sweat_smile: Anyways, it would be nice to know if there was some big update during the 2023, and what was the exact date for it.
